index.js

```diff
@@ -204,6 +204,12 @@ Wallet.fromPrivateKey = function (priv) {
   return new Wallet(priv)
 }
 
+Wallet.fromPrivateKeyString = function (priv) {
```

Comment:
Thank you! Although I am not sure this is a good idea, since there are at least 4 constructor methods, all of which would need to have a string counterpart in this case, which is a lot of extra maintenance.

Is it really necessary to handle string inputs in this library?